app/models/kaui/transaction.rb

```diff
@@ -34,7 +34,17 @@ def amount_to_money
     Kaui::Base.to_money(amount, currency)
   end
 
+
+  def processed_amount_to_money
+    Kaui::Base.to_money(processed_amount, currency)
+  end
+
   def self.amount_to_money(transaction)
     self.new(:amount => transaction.amount, :currency => transaction.currency).amount_to_money
   end
+
+  def self.processed_amount_to_money(transaction)
+    self.new(:processed_amount => transaction.processed_amount, :currency => transaction.currency).processed_amount_to_money
```

Comment:
Needs to be `:processed_currency => transaction.processed_currency` (and `processed_amount_to_money` above updated as well).

Reply:
![Screenshot 2022-10-14 at 11 04 07](https://user-images.githubusercontent.com/65641209/195808312-4b4c2c51-56a3-4b41-a9fd-9056bbf02869.png)

Right...